src/forklift/cli.py

```diff
@@ -92,10 +92,7 @@ def start_lift(file_path=None, pallet_arg=None):
     all_pallets = []
     pallets_to_lift = []
     for info in pallet_infos:
-        module_name = splitext(basename(info[0]))[0]
-        class_name = info[1]
-        log.debug('attempting to import %s from %s', info[1], info[0])
-        PalletClass = getattr(load_source(module_name, info[0]), class_name)
+        PalletClass = info[1]
```

Comment:
python does unpacking. I've been reading it is [a best practice](http://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_explicit_unpacking.html) to use. Would you replace `info[0]` and the other index accessors with explicit unpacked values like 

```py
for pallet_name, pallet_location in pallet_infos:
```

It makes readability and understandability better. I had to look pretty hard for what the info options were. This makes it much more simple to understand.